### ISSUE TITLE
MYSQL_BASIC-Eindtaak

### ISSUE THREAD

**Issue by jkhb64** [ROC-van-Amsterdam-College-Amstelland/MYSQL-BASIC#4]
Kun je deze vraag wel beantwoorden?
5. Welk spel heeft het meest geld opgebracht wereldwijd? Je weet aantal, toch niet de omzet?

Kan dit in één query? Of zijn het drie vragen?
10. Toon de gemiddelde verkoop aantallen in Noord Amerika, Europa en Japan van de spellen gemaakt door Sony Computer Entertainment voor de Playstation 4 (PS4).

**Comment by aavroc** [comment]
> 
> 
> Kun je deze vraag wel beantwoorden?
> 5. Welk spel heeft het meest geld opgebracht wereldwijd? Je weet aantal, toch niet de omzet?

Klopt, zal de vraag aanpassen zodat gevraagd wordt naar aantal  ipv opbrengst.

> Kan dit in één query? Of zijn het drie vragen?
> 10. Toon de gemiddelde verkoop aantallen in Noord Amerika, Europa en Japan van de spellen gemaakt door Sony Computer Entertainment voor de Playstation 4 (PS4).

Is 1 vraag. Zal er nog naar kijken.
